### PR TITLE
Added a monkey patch disabling _path_importer_cache

### PR DESCRIPTION
**Pull request by hoodmane**
I'm not sure if this is a good choice, but it seems like an option to consider.

We've had various troubles with the import cache and the EmscriptenFS date modified times. This completely disables the cache which should prevent any need for `importlib.invalidate_cache` and any bugs related to import cache from ever coming up again.

I'm not sure what the performance implications of this are. We could also implement a different caching strategy which is more conservative than the normal one but less conservative than always refetching from scratch.

### CODE REVIEW

File: src/pyodide.js [hunk 1]
```diff
@@ -249,12 +249,6 @@ var languagePluginLoader = new Promise((resolve, reject) => {
           }
         });
       }
-
-      // We have to invalidate Python's import caches, or it won't
-      // see the new files. This is done here so it happens in parallel
-      // with the fetching over the network.
-      self.pyodide.runPython('import importlib as _importlib\n' +
-                             '_importlib.invalidate_caches()\n');
```
Comment (hoodmane) [inline src/pyodide.js:257]: Maybe we should keep this?

**Comment by hoodmane** [comment]
@rth I think this PR is a bad idea if the existing solutions for cache problems work (what's the status of that). However, I think it would be good to keep this around as documentation if cahce problems ever come up again.

**Comment by rth** [comment]
> I'm not sure what the performance implications of this are.

You could check  `benchmarks.json` in the artifacts tab of the `benchmarks` CircleCI job and compare it to its value on master. Those benchmarks results are fairly noisy though.

**Comment by rth** [comment]
> what's the status of that

Do you mean https://github.com/iodide-project/pyodide/pull/893 ? That was merged.

> We've had various troubles with the import cache and the EmscriptenFS date modified times.

Could it be due to the above PR?

**Comment by hoodmane** [comment]
Ah no, I was referring to #737, I haven't seen this problem myself. #893 says it "Partially solves #737". What is partial about that solution?

**Comment by hoodmane** [comment]
Okay, since #737 works, this is unnecessary.

**Pull request closed without merging**